test/maconn.browser.spec.ts

```diff
@@ -11,6 +11,7 @@ describe('Multiaddr Connection', () => {
     const remoteAddr = multiaddr('/ip4/1.2.3.4/udp/1234/webrtc/certhash/uEiAUqV7kzvM1wI5DYDc1RbcekYVmXli_Qprlw3IkiEg6tQ')
     const maConn = new WebRTCMultiaddrConnection({
       peerConnection: peerConnection,
+      metrics: null,
```

Comment:
Would be nice if we explicitly tested metrics inteface?

Reply:
The [sinon-ts](https://www.npmjs.com/package/sinon-ts) module can be used to do this in a straightforward way.